demo/public/css/main.css

```diff
@@ -848,7 +848,7 @@ header {
   height: .375em;
   background: #63a7d4;
   display: block;
-  bottom: 0;
+  bottom: -20px;
```

Comment:
I generally try to avoid negative offsets, as it usually means I'm doing something wrong in the first place (and that things are even less likely to work across browsers).

That said, I don't think you're the original author of the CSS, so if it works (and you've done some light x-browser testing to verify), this is likely a-ok :).

Reply:
this was the only way I could get it to work! (the original code just used manual anchors, after I changed it to react-router <Link>s the formatting broke)